controllers/chaosimpl/stresschaos/impl.go

```diff
@@ -74,13 +74,17 @@ func (impl *Impl) Apply(ctx context.Context, index int, records []*v1alpha1.Reco
 		}
 	}
 
-	res, err := pbClient.ExecStressors(ctx, &pb.ExecStressRequest{
+	req := pb.ExecStressRequest{
 		Scope:           pb.ExecStressRequest_CONTAINER,
 		Target:          containerId,
 		CpuStressors:    cpuStressors,
 		MemoryStressors: memoryStressors,
 		EnterNS:         true,
-	})
+	}
+	if stresschaos.Spec.Stressors.MemoryStressor != nil {
+		req.OomScoreAdj = int64(stresschaos.Spec.Stressors.MemoryStressor.OOMScoreAdj)
```

Comment:
Why use `int64` here? I found that everything else uses `int` except for proto.

Reply:
Because the size of `int` depends on the implementation but is not part of the go language, the `gRPC` only allows using `int32` or `int64`.

But anyway, I should use `int32` here. I have submitted a commit to turn it into `int32`.